Debug docker images fail to build: inconsistent vendoring in /go/src/github.com/go-delve/delve

E.g. #3187 is failing with this error:
```
#15 [linux/amd64 build 3/3] RUN if [[ "amd64" == "s390x"  ||  "amd64" == "ppc64le" ]] ; then 	touch /go/bin/dlv;     else         go get github.com/go-delve/delve/cmd/dlv &&         cd /go/src/github.com/go-delve/delve &&         make install;     fi
#15 19.99 go: inconsistent vendoring in /go/src/github.com/go-delve/delve:
#15 19.99 	github.com/derekparker/trie@v0.0.0-20200317170641-1fdf38b7b0e9: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
#15 19.99 
#15 19.99 run 'go mod vendor' to sync, or use -mod=mod or -mod=readonly to ignore the vendor directory
#15 19.99 make: *** [Makefile:28: install] Error 1
#15 ERROR: process "/bin/sh -c if [[ \"$TARGETARCH\" == \"s390x\"  ||  \"$TARGETARCH\" == \"ppc64le\" ]] ; then \ttouch /go/bin/dlv;     else         go get github.com/go-delve/delve/cmd/dlv &&         cd /go/src/github.com/go-delve/delve &&         make install;     fi" did not complete successfully: exit code: 2
```

--- a/docker/Makefile
+++ b/docker/Makefile
@@ -1,7 +1,7 @@
 VERSION := 1.0.0
 ROOT_IMAGE ?= alpine:3.13
 CERT_IMAGE := $(ROOT_IMAGE)
-GOLANG_IMAGE := golang:1.15-alpine
+GOLANG_IMAGE := golang:1.16-alpine
 
 BASE_IMAGE := localhost:5000/baseimg_alpine:latest
 DEBUG_IMAGE := localhost:5000/debugimg_alpine:latest

--- a/docker/debug/Dockerfile
+++ b/docker/debug/Dockerfile
@@ -9,9 +9,7 @@ RUN apk add --update --no-cache ca-certificates make git
 RUN if [[ "$TARGETARCH" == "s390x"  ||  "$TARGETARCH" == "ppc64le" ]] ; then \
 	touch /go/bin/dlv; \
     else \
-        go get github.com/go-delve/delve/cmd/dlv && \
-        cd /go/src/github.com/go-delve/delve && \
-        make install; \
+        go install github.com/go-delve/delve/cmd/dlv@latest; \
     fi
 
 FROM $golang_image

